themes/bootstrap3/templates/myresearch/fines.phtml

```diff
@@ -67,22 +67,28 @@
     ?>
     <table class="table table-striped">
       <caption class="sr-only"><?=$this->transEsc('Your Fines')?></caption>
+      <thead>
       <tr>
         <?php foreach ($columns as $header): ?>
-          <th><?=$this->transEsc($header)?></th>
+          <th scope="col"><?=$this->transEsc($header)?></th>
         <?php endforeach; ?>
       </tr>
+      </thead>
+      <tbody>
       <?php for ($row = 0; $row < $rowCount; $row++): ?>
         <tr>
           <?php foreach ($columns as $column): ?>
             <td><?=$tableData[$column][$row]?></td>
           <?php endforeach; ?>
         </tr>
       <?php endfor; ?>
-      <tr style="font-weight:bold">
-        <td colspan="<?=count($columns) - 1?>"><?=$this->transEsc('Total Balance Due')?></td>
-        <td><?=$this->safeMoneyFormat($totalDue / 100.00) ?></td>
+      </tbody>
+      <tfoot>
+      <tr>
+        <th colspan="<?=count($columns) - 1?>"><?=$this->transEsc('Total Balance Due')?></th>
```

Comment:
Should this have a scope of row, or is that implied?

Reply:
It should, yes... and actually, the fines' names should probably be scoped headers as well. I will amend that!